functions/ldap.js

```diff
@@ -14,6 +16,8 @@ exports.plugin = {
   name: 'ldap',
   register: async function (server) {
     server.ext('onRequest', (request, h) => {
+      helpers.setQueryParameterTemplateValues(request.query, config.ldap);
```

Comment:
```suggestion
      helpers.setQueryParameterTemplateValues(request.query, config.ldap);
```

Is `request.query` at this point an object or still the serialized string value?

Reply:
request.query is already an object at this point